scripts/pi-hole/js/list.js

```diff
@@ -146,7 +146,8 @@ function add(arg) {
         method: "post",
         data: {"domain":domain.val().trim(), "list":locallistType, "token":token},
         success: function(response) {
-          if (response.indexOf("] Pi-hole blocking is ") === -1) {
+          if (locallistType !== "wild" && response.indexOf("] Pi-hole blocking is ") === -1 ||
+              locallistType === "wild" && response.length > 1) {
```

Comment:
Due to short-circuiting boolean logic, the AND statements need to be put in parentheses so even if the first check fails, it will run the checks after the OR symbol.

Reply:
Though, it appears that JS handles this correctly...